Objects/longobject.c

```diff
@@ -4498,7 +4551,7 @@ long_invert(PyLongObject *v)
     PyLongObject *x;
     if (IS_MEDIUM_VALUE(v))
         return _PyLong_FromSTwoDigits(~medium_value(v));
-    x = (PyLongObject *) long_add(v, (PyLongObject *)_PyLong_GetOne());
+    x = (PyLongObject *) long_add(v, DIGIT_ONE);
```

Comment:
This and similar changes should be reverted - they're not directly related to the purpose of the PR, and there were good reasons for introducing `_PyLong_GetOne`. (@vstinner is in a better position to articulate those reasons than I am)

Reply:
Reverted.